Tools/TestWebKitAPI/Tests/WebKitCocoa/audio-context-playing.html

```diff
@@ -4,15 +4,37 @@
 <script>
 var context;
 var oscillator;
+var scriptProcessor;
+var isFirst = true;
+
 function startPlaying()
 {
     if (context)
         context.close();
 
-    context = new AudioContext();
-    oscillator = new OscillatorNode(context);
-    oscillator.connect(context.destination);
-    oscillator.start();
+    isFirst = true;
+    return new Promise(function(resolve, reject) {
```

Comment:
Do we have a similar danger of flakiness from generateAudioInMediaStreamTrack()?